docs/smartcontracts.md

```diff
@@ -106,7 +85,7 @@ values={[
 // const Tezos = new TezosToolkit('https://ghostnet.ecadinfra.com');
 
 Tezos.contract
-  .at('KT1BJadpDyLCACMH7Tt9xtpx4dQZVKw9cDF7')
+  .at('KT1RMvFVQ3TT2MWme6VwdsQgYR3BjKnWt5iZ')
```

Comment:
this contract is also for the michelson contract 
its missing the reset entrypoint
could either remove the entrypoint or make consistent with these contracts

Reply:
fixed with correct new address